backend/src/utils/scratchpad.py

```diff
@@ -0,0 +1,13 @@
+scratchpad = []
+
+
+def get_scratchpad() -> list:
+    return scratchpad
+
+
+def update_scratchpad(agent_name, task, result):
+    scratchpad.append({"agent_name": agent_name, "task": task["summary"], "result": result})
+
+
+def clear_scratchpad():
+    scratchpad.clear()
```

Comment:
nice! Python question from me, will only one scratchpad instance ever be created, regardless of how many things import this file?

Reply:
We don't import the variable, just the methods. Therefore, I believe all method calls with access the same scratchpad, regardless of where they're imported to. So I'm gonna say yes, but only because I don't see how we would be "newing up" a new scratchpad variable without importing the variable itself anywhere

Reply:
sounds good to me. Yeah my only concern was if the whole script `scratchpad.py` is run every time we import one of the methods, but sounds like that is not the case